modules/kubernetes/logs/masks/all.river

```diff
@@ -0,0 +1,94 @@
+/*
+Module: mask-all
+Description: Wrapper module to include all masking modules
```

Comment:
Other note is instead of `logs` for pathing if we should go with loki since it maps to the type of component being used. @erikbaranowski thoughts here?

Reply:
@mattdurham @erikbaranowski  are you saying that the paths should be:

```
modules/kubernetes/loki/...
modules/kubernetes/mimir/...
``` 

I'm not sure I'd agree that we should map to the component being used, I'd rather map to the type of entity being consumed / processed by the agent rather than the destination component.  I.e. metrics, logs, traces, etc.  Thoughts?

Reply:
The main argument I have against it is that we also support OTEL so things like logs and metrics are harder to differentiate in that case.

Reply:
Conversed a bit out of band and this is something we can always clean up later if we feel we need to.